pom.xml

```diff
@@ -46,6 +73,10 @@
             <id>repsy</id>
             <url>https://repo.repsy.io/mvn/elytraforce/default</url>
         </repository>
+        <repository>
+            <id>snapshots</id>
+            <url>https://repo.repsy.io/mvn/elytraforce/snapshots</url>
```

Comment:
I don't suggest depending on snapshot dependencies except during development. Released software should use stable (non-SNAPSHOT) dependencies